### PR TITLE
Fix tests for API28-33

### PR DESCRIPTION
**Pull request by LinX64**

### CODE REVIEW

**Comment by LinX64** [comment]
[Fix tests | 28-33 API new](https://trello.com/c/XGgdVXe5/1710-fix-tests-28-33-api-new)

**Comment by LinX64** [comment]
[CircleCI | Update](https://trello.com/c/vVYlTbQc/1711-circleci-update)

**Comment by LinX64** [comment]
[S&F > map: crash on session view](https://trello.com/c/EvxiAZNh/1712-sf-map-crash-on-session-view)

File: app/src/androidTest/java/pl/llp/aircasting/SearchFollowTest.kt [hunk 1]
```diff
@@ -287,29 +282,56 @@ class SearchFollowTest {
...
 
+    @Test
+    fun secondGoToFollowingTab_andUnfollow_the_followed_Session() {
+        mainActivityScenario = ActivityScenario.launch(startIntent)
+
+        unFollowFollowedSessionFromTheFollowingTab()
+
+        mainActivityScenario.close()
+    }
+
```
Comment (xameYeoh) [inline app/src/androidTest/java/pl/llp/aircasting/SearchFollowTest.kt:293]: If this is a test, I think there should be at least some sort of assertion and check going on here, it looks more like a helper function

**Review by xameYeoh — APPROVED** [review]

File: app/src/androidTest/java/pl/llp/aircasting/SearchFollowTest.kt (hunk 1 above)
Comment (LinX64) [reply app/src/androidTest/java/pl/llp/aircasting/SearchFollowTest.kt:293]: That's right, but we either have to change the logic for the followed session tests or unfollow it to make it work. So I guess we can rely on this for now.

**Pull request merged**